src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -143,6 +171,19 @@
      */
     public static final String MSG_KEY = "array.trailing.comma";
 
+    /**
+     * Control whether to always check for a trailing comma, even when an array is inline.
+     */
+    private boolean alwaysDemandTrailingComma;
+
+    /**
+     * Setter to control whether to always check for a trailing comma, even when an array is inline.
+     * @param alwaysDemandTrailingComma user's value
```

Comment:
please add a new line 
```java
/**
 * Setter to control ...
 *
 * @param alwaysDemandTrailingComma ...
 */
```
also I wonder about instead of `user's value`, maybe restating `whether to always check for a trailing comma`

Reply:
Done.